src/SceneGenerator.js

```diff
@@ -0,0 +1,32 @@
+/*Object used for generating and presenting visual scenes*/
+var SceneGenerator = function(filterName, width, height) {
```

Comment:
Often indenting issues is because of tabs and spaces. There is a handy tool in sublime to convert all tabs to spaces easily. Just press ctrl+shift+p then search for "spaces". Convert tabs to spaces should be the first one.